target/linux/octeon/base-files/lib/upgrade/platform.sh

```diff
@@ -1,6 +1,7 @@
 #
 # Copyright (C) 2014 OpenWrt.org
 #
+RAMFS_COPY_BIN=mkfs.f2fs
```

Comment:
Will this affect implementation of erlite somehow?

Reply:
No.  In fact, since the ER uses f2fs, I'm surprised it wasn't already included.